package.json

```diff
@@ -9,9 +9,10 @@
     "compile": "webpack",
     "karma": "karma start karma.config.js",
     "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint '**/*.js' --fix",
     "prepublish": "yarn compile",
     "start": "webpack-dev-server",
-    "test": "yarn ci:lint && yarn ci:test",
+    "test": "yarn ci:test && yarn ci:lint",
```

Comment:
How would you feel about adding lint-fix to the beginning, then? Not even being able to see my test failures because I didn't pad my blocks is a really irritating dev flow.